pkgs/servers/owncast/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildGoModule, fetchFromGitHub, bash, which, ffmpeg, makeWrapper, ... }:
+
+buildGoModule rec {
+
+  pname = "owncast";
+  version = "0.0.8";
+
+  src = fetchFromGitHub {
+    owner = "owncast";
+    repo = "owncast";
+    rev = "v${version}";
+    sha256 = "0md4iafa767yxkwh6z8zpcjv9zd79ql2wapx9vzyd973ksvrdaw2";
+  };
+
+  vendorSha256 = "sha256-bH2CWIgpOS974/P98n0R9ebGTJ0YoqPlH8UmxSYNHeM=";
+
+  propagatedBuildInputs = [ ffmpeg ];
+
+  buildInputs = [ makeWrapper ];
+
+  postInstall = ''
```

Comment:
You should include the `static/` and `webroot/` directories in the package as they are needed at runtime.